templates/operations.ts.hbs

```diff
@@ -180,8 +180,15 @@
       const response = await fetch(req);
       if (rawResponse) {
         return response;
-      } else {
-        return response.json();
+      } else if (response.ok || response.status === 304) {
+        {{#unless (eq (getReturnTypeFromOperation this) "void") }}
```

Comment:
This if block will be empty when the return type is `void`. I don't think the linter will like that.

Reply:
I guess `return; ` is fine?

Reply:
The entire `else` block should be inside the `{{#unless}}`, otherwise you'll end up with
```ts
else { }
```